frontend/src/components/c-summary-charts.vue

```diff
@@ -66,7 +66,8 @@
             span.tooltip-text Click to view group's code
         a(
           onclick="deactivateAllOverlays()",
-          v-on:click="openTabZoom(repo[0], filterSinceDate, filterUntilDate, isGroupMerged(getGroupName(repo)))"
+          v-on:click="openTabZoom(JSON.parse(JSON.stringify(repo[0])), filterSinceDate, filterUntilDate,\
```

Comment:
I think it was not addressed earlier, but any reason why you decided to stringify and parse again?

Reply:
This was to be consistent with the previous functionality of toggles being reset when switching between users (if we pass without deep copying, toggling the visibility in the zoom panel will change the commits in the underlying `user` object, and so will persist after switching)

Reply:
I see, would it be possible to move the stringify and parse into the `openTabZoom` function and add a comment stating its purpose? I think it was not immediately clear from reading the code.

Reply:
Just pushed a commit for this!